rpcs3/Crypto/unself.cpp

```diff
@@ -5,10 +5,11 @@
 #include "Emu/FS/vfsLocalFile.h"
 #include "unself.h"
 #pragma warning(push)
-#pragma message("TODO: remove wx dependencies: <wx/mstream.h> <wx/zstream.h>")
+#pragma message("TODO: remove wx dependencies: See comment below.")
 #pragma warning(disable : 4996)
-#include <wx/mstream.h>
-#include <wx/zstream.h>
+
+// TODO: Still reliant on wxWidgets for zlib functions. Alternative solutions?
+#include <wxWidgets/src/zlib/zlib.h>
```

Comment:
I think this should either be <zlib.h> or "wxWidgets/src/zlib/zlib.h" 
See http://stackoverflow.com/a/21594